example/notify_inval_entry.c

```diff
@@ -324,6 +347,9 @@ int main(int argc, char *argv[]) {
 
     fuse_daemonize(opts.foreground);
 
+    // needed to correctly exit fuse_session_loop()
```

Comment:
No, this is needed to sent a signal to the main thread so it gets worked up and notices that it should exit. It is not about how to exit, but when to exit.